src/scripts.js

```diff
@@ -294,7 +294,7 @@ function filterFavoriteRecipesByName(name) {
 
 function showFavoriteRecipeImages(){
   favoriteRecipeImages.innerHTML = '';
-  favoriteRecipes.forEach(recipe => {
+  user.recipesToCook.forEach(recipe => {
```

Comment:
might run into another conflict here, i refactored this function in my PR because it wasnt working quite right